package.json

```diff
@@ -7,7 +7,8 @@
     "build/*"
   ],
   "dependencies": {
-    "commander": "^8.1.0"
+    "commander": "^8.1.0",
+    "commander-ts": "^0.1.5"
```

Comment:
not officially reviewing this, but just peeked at it. We shouldn't need commander-ts dependency